src/components/Calculator.jsx

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from 'react';
 import calculate from '../logic/calculate';
 import AnswerSection from './AnswerSection';
 import keysPress from '../logic/keysPress';
+import QuoteApi from './QuoteApi';
```

Comment:
As per the project requirements, the `Quotes` component should be imported into the main component which is `App.js`, and not in the `Calculator` component. Since the `Quotes` component is totally independent of the `Calculator` component, it would make much more sense if it was in the `App.js` component.